### DISCUSSION TITLE
New chromedriver Version: 120.0.6099.71 (r1217362)

### DISCUSSION THREAD

**Discussion by Moiseosanchez** [ultrafunkamsterdam/undetected-chromedriver#1699] [Q&A]
Hi! First of all, thanks for this magnificent project. 

Today doesn't work with chromedriver Version: 120.0.6099.71 

Could we do something to fix that? Maybe if we change something we can use new chromedriver versions or do you have to update something in the lib?

Cheers!

**Comment by markthepixel** [comment]
Agreed. or a easier way to hardset the version based on Google's servers or something
